README.md

```diff
@@ -969,20 +969,23 @@ meetingSession.audioVideo.realtimeSubscribeToAttendeeIdPresence(
 
 ### Monitoring and alerts
 
-**Use case 25.** Add an observer to receive video metrics. See `AudioVideoObserver` for more available metrics,
-such as WebRTC statistics processed by Chime SDK.
+**Use case 25.** Add an observer to receive WebRTC metrics processed by Chime SDK such as bitrate, packet loss, and bandwidth. See `AudioVideoObserver` for more available metrics.
 
 ```js
 const observer = {
-  videoSendHealthDidChange: (bitrateKbps, packetsPerSecond) => {
-    console.log(`Sending bitrate in kilobits per second: ${bitrateKbps} and ${packetsPerSecond}`);
+  metricsDidReceive: (clientMetricReport) => {
+    const metricReport = clientMetricReport.getObservableMetrics();
+
+    const videoUpstreamPacketPerSecond = metricReport.videoPacketSentPerSecond;
+    const videoUpstreamBitrate = metricReport.videoUpstreamBitrate;
+    const availableSendBandwidth = metricReport.availableOutgoingBitrate;
+    const availableRecvBandwidth = metricReport.availableIncomingBitrate;
+    const audioSpeakerDelayMs = metricReport.audioSpeakerDelayMs;
+
+    console.log(`Sending video bitrate in kilobits per second: ${videoUpstreamBitrate / 1000} and ${videoUpstreamPacketPerSecond}`);
```

Comment:
nit: Do we need to mention that `${videoUpstreamPacketPerSecond}` is video packet per second?

Reply:
Yeah we should, updated